### DISCUSSION TITLE
How do I set up Ros2 for Raspberry Pi 4 when link does not work anymore?

### DISCUSSION THREAD

**Discussion by ctgitxb** [iRobotEducation/create3_docs#136] [Questions] [answered]
Under set up compute boards in create3 docs:
If you click on  
[Installing ROS 2 on Ubuntu Linux](https://docs.ros.org/en/galactic/Installation/Ubuntu-Install-Binary.html)[1](https://iroboteducation.github.io/create3_docs/setup/pi4/#fn:1) - official Open Robotics documentation
 you will get the following

Not Found
The requested URL /en/galactic/Installation/Ubuntu-Install-Binary.html was not found on this server.

How do I set up galactic Ros2 when this link does not work?

**Comment by shamlian** [comment, marked as answer]
That's not our documentation, but it's possible to get Ubuntu 20.04 on an SD card using the Raspberry Pi imager at https://www.raspberrypi.com/software/ , and then install ROS 2 by following the step-by-step instructions at https://iroboteducation.github.io/create3_docs/setup/pi4/#step-by-step .

**Reply by ctgitxb** [reply]
Thank you for your response.
Yes I think I can get Ubuntu on the Raspberry Pi.  However, how do I install  Ros2 with no instructions?

**Reply by shamlian** [reply]
_Oh_, I see. You can follow the instructions on our page verbatim: 
https://iroboteducation.github.io/create3_docs/setup/pi4/#step-by-step .

I'll have to see what happened to OR's docs.

**Reply by shamlian** [reply]
Looks like OR's Ubuntu install docs are now found at https://docs.ros.org/en/galactic/Installation/Ubuntu-Install-Debians.html or perhaps https://docs.ros.org/en/galactic/Installation/Alternatives/Ubuntu-Install-Binary.html; I will update the iRobot docs site. Thanks. You should be good to go if you just follow the step-by-step on the page that you originally navigated over there from.

**Reply by ctgitxb** [reply]
Thank you!

**Reply by shamlian** [reply]
You're welcome; if you are successful, please let us know by marking the top level post as the answer (and if not, please write again! :) ).
